color

```diff
@@ -0,0 +1,18 @@
+CSS OFF top background #ccc9dc
```

Comment:
I'd suggest placing this file in a dir called `resources` or `docs` or something along those lines.